### PR TITLE
Release forked 2 7 1

### PR DESCRIPTION
**Pull request by erkiesken**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "build": "node-gyp build",
     "test": "make test",
     "install": "node-gyp rebuild",
-    "prepack": "node ./ci/prepublish.js"
+    "_prepack": "node ./ci/prepublish.js"
```
Comment (erkiesken) [inline package.json]: This was messing with package version while built in a brach, shoudl remove `_` before merging back to master.

**Comment by poolik** [comment]
@erkiesken should probably close this pr, not needed anymore

**Comment by erkiesken** [comment]
Probably can also delete the repo now.

**Comment by poolik** [comment]
@erkiesken we very much use the repo as still using our fork:
https://github.com/6fold/messaging-utils/pull/28

**Pull request closed without merging**